### PR TITLE
Update codeowners

### PR DESCRIPTION
**Pull request by calebmshafer**
- Remove the requirement that one person approves all changelogs. Allow pkg owners to review them
- Add a second reviewer to the imodel-select-react pkg.

@wilmaier realized you were the only one on the imodel-select.  @aruniverse or @JoeZman you guys okay with being added here? If not, I can handle it.

### CODE REVIEW

**Comment by aruniverse** [comment]
Dont love taking ownership, but this will do for now. I dont imagine we will have much to do for this pkg. This will also get us around annoying Wil.

**Comment by calebmshafer** [comment]
Sounds good.  I'll just leave myself for now so that we have at least 2 people.

@bsteinbk I think you initially wrote this right, mind being a reviewer for it?  Hopefully things aren't needed too often :)

**Comment by calebmshafer** [comment]
@aruniverse @jason-crow I just made an update to not require that I approve all changelogs as well.  This should now be on the owners of the packages.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -9,7 +9,15 @@
...
+
+/packages/imodel-select  @wilmaier @calebmshafer
+/common/changes/@bentley/imodel-select-react  @wilmaier @calebmshafer
+
 /packages/markup-frontstage  @JoeZman @archana-maharjan
-/packages/tree-widget  @JoeZman @diegopinate @aruniverse @JValiunas @michaalx
+/common/changes/@bentley/markup-frontstage-react @JoeZman @archana-maharjan
+
 /packages/property-grid  @JoeZman @aruniverse
+/common/changes/@bentley//property-grid  @JoeZman @aruniverse
```
Comment (aruniverse) [inline .github/CODEOWNERS:20]: should be: `/common/changes/@bentley/property-grid`

Comment (aruniverse) [reply .github/CODEOWNERS:20]: you have an extra `/`

Comment (aruniverse) [reply .github/CODEOWNERS:20]: also, can we add diego as a code owner for this one?

Comment (calebmshafer) [reply .github/CODEOWNERS:20]: I had fixed that but forgot to save the file before I committed :)  Pushing now.

Comment (calebmshafer) [reply .github/CODEOWNERS]: Add @diegopinate and fixes the path issue.

**Pull request merged**